packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox-group/example.html

```diff
@@ -0,0 +1,180 @@
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example 3 with Group Help text</h2>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta>
+    <label slot="group-label">Food</label>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="pizza" />
+      <label slot="label">Pizza</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="food-1"
+        value="sandwich"
+        disabled
+      />
+      <label slot="label">Sandwich</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="sushi" />
+      <label slot="label">Sushi</label>
+      <span slot="error">Subject to availibility</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="spaghetti" />
+      <label slot="label">Spaghetti</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+    <span slot="group-help">This is a group help message</span>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example 4 Disabled group</h2>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta disabled>
+    <span slot="group-label">Food</span>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="pizza" />
+      <label slot="label">Pizza</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="food-1"
+        value="sandwich"
+        disabled
+      />
+      <label slot="label">Sandwich</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="sushi" />
+      <label slot="label">Sushi</label>
+      <span slot="error">Subject to availibility</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="spaghetti" />
+      <label slot="label">Spaghetti</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<gux-button-slot>
+  <button
+    type="button"
+    onclick="(function () {
+      document.querySelector(`input[name='food-1[]'][value='hamburger']`).indeterminate = true;
+    })()"
+  >
+    Set Hamburger field to indeterminate
+  </button>
+</gux-button-slot>
+
+<gux-button-slot>
+  <button
+    type="button"
+    onclick="(function () {
+      const field = document.querySelector(`input[name='food-1[]'][value='pizza']`)
+      field.checked = !field.checked;
+    })()"
+  >
+    Toggle Pizza field
+  </button>
+</gux-button-slot>
```

Comment:
I think the selectors on these buttons need to be updated as I couldn't toggle them.